src/components/form/Form.vue

```diff
@@ -567,14 +567,22 @@ export default {
         },
         getHiddenFields() {
             const hiddenInputFields = this.$el.querySelectorAll('input[type=hidden]');
-            const fieldData = {
-            };
 
-            hiddenInputFields.forEach((field) => {
-                fieldData[field.name] = field.value;
-            });
+            return [...hiddenInputFields].map(this.parseBooleanStringsFromInputField);
+        },
+        parseBooleanStringsFromInputField(inputField) {
+            let value = inputField.value;
 
-            return fieldData;
+            if (value === 'true') {
```

Comment:
This could probably written a bit neater with a ternary statement

Reply:
Great shout :)

Reply:
@GemmaLouise - I remember I did attempt this at some point but was encountering `ESLint: Do not nest ternary expressions.(no-nested-ternary`.